website/community/committers/release.md

```diff
@@ -99,6 +99,7 @@ This issue is used to track tasks of the opendal ${opendal_version} release.
   - [ ] java
   - [ ] nodejs
 - [ ] Update docs
+- [ ] Update dependencies
```

Comment:
With deps checked in CI, we don't need this step anymore.

Reply:
dep check and dep generate are different. We can perform them together during release for now.

Reply:
Added check in CI.